INSTALLATION.md

```diff
@@ -0,0 +1,156 @@
+# Walkthrough 
+
+This is a walkthrough for the OCP worker on VMs project, that can be found here: [ocp-lab-setu]() https://github.com/oVirt/ocp-lab-setup)
+
+## Setting up the environment
+
+### Get an inventory
+```
+./05-get-inventory.sh http://quads.rdu2.scalelab.redhat.com/cloud/cloud09_ocpinventory.json
+```
+We removed the first 5 entries, since we need those hosts as masters and infra nodes which resulted in the `hosts` file.
+```
+e21-h20-b03-fc640.rdu2.scalelab.redhat.com,eno1,ens2f0,ens2f1
+e21-h20-b04-fc640.rdu2.scalelab.redhat.com,eno1,ens2f0,ens2f1
+```
+We need to distribute our public key so we can work on the nodes without the need to authenticate every time. Make sure it uses the correct key in case you have more than just one in the user's .ssh directory.
+
+```
+./10-ssh-keys.ssh
+```
+This copies your ssh key ~/.ssh/id_rsa.pub key to the nodes in the `hosts` file.
+
+Now we need to install the required packages on all the nodes, the engine as well as the VM hosts:
+```
+./15-install-packages.sh http://bob.eng.lab.tlv.redhat.com/builds/4.5/rhv-4.5.3-1/api/ 
```

Comment:
again no internal FQDNs please

Reply:
that's a tricky one, but you're right.
removed